start/sw/gui-app/gtk.md

```diff
@@ -0,0 +1,60 @@
+---
+title: GTK+
+subsection: gui-app
+section: start-sw
+description: 
+---
+
+# Create an application using GTK+
+
+GTK+ is a cross platform graphical toolkit which stands for GIMP toolkit.
+It was started in 1998 as an free and open source alternative to Qt which was
+proprietary at the time. GTK+ is published under the GNU LGPL 2.1 license.
+
+## Installing the development packages
+
+GTK+ is written in C so the minimum required to create a GTK+ application is a
+C compiler and the GTK+ development package:
+
+```
+$ sudo dnf install gcc gtk3-devel
```

Comment:
Maybe it would be a good  idea to point to C app guide we have, or cli-app quickstart:

 - http://developer.fedorainfracloud.org/tech/languages/c/c_installation.html
 - http://developer.fedorainfracloud.org/start/sw/cli-app/about.html